controller/api/models.py

```diff
@@ -1095,8 +1096,9 @@ def _log_domain_removed(**kwargs):
 
 
 def _log_cert_added(**kwargs):
-    cert = kwargs['instance']
-    logger.info("cert {} added".format(cert))
+    if kwargs.get('created'):
+        cert = kwargs['instance']
+        logger.info("cert {} added".format(cert))
```

Comment:
Just to note here: there was an early convention in the Deis python codebase of using double quotes `"` for format strings but single quotes `'` for all others. This was a nudge toward common standards in a polyglot codebase for previous products built with Ruby and CoffeeScript. 

So that's why you'll see this difference in much of the Deis Python code, but it may not be productive to bother about it now. :snake: